src/tls/Makefile

```diff
@@ -1,32 +1,25 @@
 EXTRACT='Nothing'
-FLAVOR=generic
+FLAVOR?=Model
 
-all:
-	rm -f .depend.OCaml .depend.Kremlin
-	$(MAKE) ocaml-all kremlin-all
-
-clean-depend:
-	rm -rf .depend.OCaml .depend.Kremlin
-
-ocaml-all kremlin-all: clean-depend
-
-include Makefile.common
+all: ocaml-all kremlin-all
```

Comment:
this removes an important feature, which is to blast all the .depend files by default when entering the build via the top-level Makefile; we much rather prefer wasting a couple seconds rather than having a substantial amount of build errors whose fix is "remove your .depend"

Reply:
Yes, I decided against that on purpose. For instance, re-computing the dependencies is annoying when emacs wants to look up the `*.fst-in` options. Instead, `.depend` files now have dependencies too. For CI, I agree, that should always be recomputed.

Reply:
On the other hand, that doesn't work while modifying the files during active work... We can just as well add it back I guess.